### ISSUE TITLE
Seeing error about Maven class missing under 0.5.0

### ISSUE THREAD

**Issue by abrooksv** [JetBrains/gradle-intellij-plugin#542]
I tried 0.5.0, but sometimes the build would fail due to kotlinCompile error:

```
Options for KOTLIN DAEMON: IncrementalCompilationOptions(super=CompilationOptions(compilerMode=INCREMENTAL_COMPILER, targetPlatform=JVM, reportCategories=[0], reportSeverity=2, requestedCompilationResults=[0]kotlinScriptExtensions=[kt, kts, java]), areFileChangesKnown=false, modifiedFiles=null, deletedFiles=null, workingDir=/Users/<user>/aws-intellij-toolkit/jetbrains-core/build/kotlin/compileKotlin, multiModuleICSettings=MultiModuleICSettings(buildHistoryFile=/Users/<user>/aws-intellij-toolkit/jetbrains-core/build/kotlin/compileKotlin/build-history.bin, useModuleDetection=false), usePreciseJavaTracking=trueoutputFiles=[/Users/<user>/aws-intellij-toolkit/jetbrains-core/build/classes/kotlin/main, /Users/<user>/aws-intellij-toolkit/jetbrains-core/build/kotlin/compileKotlin])
e: Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class org.jetbrains.idea.maven.model.MavenArtifact, unresolved supertypes: org.jetbrains.idea.maven.model.MavenCoordinate

> Task :jetbrains-core:compileKotlin FAILED
```

There were not any more logs in `--info`.

It did not seem to be deterministic, my coworker saw it until he cleaned first, but I saw it more frequently.

Revert PR: https://github.com/aws/aws-toolkit-jetbrains/pull/2133

Any ideas?

**Comment by zolotov** [comment]
Hi,

> Any ideas?

No, not really. Probably the problem in kotlin compiler. I don't really understand why it even tries to compile those classes. You didn't you add a dependency on intellij maven plugin, right?

**Comment by abrooksv** [comment]
Ya we do have a dependency on "org.jetbrains.idea.maven"

**Comment by abrooksv** [comment]
I can't seem to reproduce it now and I am on a new laptop

**Comment by abrooksv** [comment]
Not seeing this anymore, will re-open if I see it and can triage it more

**Comment by zolotov** [comment]
will be fixed in 0.6.3
